pages/apis/api_differences.md.erb

```diff
@@ -0,0 +1,48 @@
+# API Differences Between REST and GraphQL
+
+Buildkite provides both a [REST API](https://buildkite.com/docs/apis/rest-api) and [GraphQL API](/docs/apis/graphql-api), but there is no full parity between the two. Certain actions can only be performed using the REST API, but not GraphQL, and vice versa.
+
+{:toc}
+
+The strengths of the GraphQL API are in complex data queries, and the strengths of the REST API are in creating and modifying records.
+
+On this page we've collected the known limitation where some API features are only available with either REST or GraphQL.
+
+## Features only available in the REST API
+
+* **ANNOTATIONS:** [retrieve and list annotations](/docs/apis/rest-api/annotations) on a build, but not create them (however, you can create the annotations using the [buildkite-agent annotate command](/docs/agent/v3/cli-pipeline) from within a job).
+* **BUILDS:** set [`branch_configuration` and `github provider configuration`](/docs/pipelines/branch-configuration#step-level-branch-filtering) on pipelines.
+* **BUILDS:** [get a build by ID](/docs/apis/rest-api/builds#get-a-build).
+* **JOBS:** [get an output of job logs](/docs/apis/rest-api/jobs#get-a-jobs-log-output).
+* **JOBS:** [retry data for jobs](/docs/apis/rest-api/jobs#retry-a-job).
+* **META:** [get a list of IPs from which Buildkite sends webhooks](/docs/apis/rest-api/meta#get-meta-information).
+* **PIPELINES:** [set provider properties](https://bk-docs-pr-1613.onrender.com/docs/apis/rest-api/pipelines#provider-settings-properties) `provider_settings` allow configuring how the pipeline is triggered based on the source code provider's events; available on pipeline for all the pipeline inputs on pipeline create.
+
+## Features only available in the GraphQL API
+
+* **AGENTS:** [get a list of agent token IDs](/docs/apis/graphql/graphql-cookbook#get-a-list-of-agent-token-ids).
```

Comment:
“Agent tokens” is generally only available via graphql. (Agent token improvements is on the roadmap, and will likely include some work on the APIs.)